RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -1,18 +1,20 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoJets.JetProducers.PileupJetIDParams_cfi import * 
+from RecoJets.JetProducers.PileupJetIDParams_cfi import *
 
 #_stdalgos_4x = cms.VPSet(full,   cutbased,PhilV1)
 _stdalgos_5x = cms.VPSet(full_5x,cutbased,PhilV1)
 
-#_chsalgos_4x = cms.VPSet(full,   cutbased) 
+#_chsalgos_4x = cms.VPSet(full,   cutbased)
 _chsalgos_5x = cms.VPSet(full_5x_chs,cutbased)
 _chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
 _chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
 _chsalgos_81x = cms.VPSet(full_81x_chs,cutbased)
+_chsalgos_94x = cms.VPSet(full_94x_chs,cutbased)
+_chsalgos_102x = cms.VPSet(full_102x_chs,cutbased)
 
-_stdalgos    = _chsalgos_81x
+_stdalgos    = _chsalgos_102x
```

Comment:
this will lead to changes in physics results, which is not allowed in production.
The goal is to enable new items with specific modifiers aiming for production.
I could suggest to use run2_miniAOD_devel, but some more specific nanoAODv7 modifier is more appropriate in this case.

@peruzzim what are you planning to use to enable new features for nanoAODv7 campaign?

Reply:
will be set explicitly in the NanoAOD configuration, should stay to the old default here and protected by run2_miniAOD_devel as you suggested

Reply:
sorry but I think I am a bit confused, should I change something else to change this explicitely in the nanoAOD config, or it is something that @peruzzim will do it?

Reply:
this should be:
```
_stdalgos    = _chsalgos_81x
run2_miniAOD_devel.toReplaceWith(_stdalgos, _chsalgos_102x)
```

and I'll then do the right customization in the NanoAOD configuration in a subsequent PR